packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_ui_components/update_button.dart

```diff
@@ -23,7 +23,7 @@ class UpdateButton extends StatelessWidget {
         icon: Icon(context.designSystem.icons.reload),
         tooltip: context.l10n.reload,
         color: _isActive
-            ? context.designSystem.colors.activeButtonTextColor
+            ? context.designSystem.colors.appBarIconColor
```

Comment:
Is the `UpdateButton` only part of the app bar? If yes than we may need to ensure that all app bar icons will use this color. If not, we need to update the color name.

Reply:
Yes, this `UpdateButton` is only used in the AppBar currently. You're right, it may be better to rename the color to something like `updateIconAppBarColor` unless we add more components which use it